### ISSUE TITLE
ipypopout doesn't work when solara sub-mounted into fastapi app

### ISSUE THREAD

**Issue by havok2063** [widgetti/solara#482]
The url for the popout has the incorrect base path.  Starting the fastapi app, and clicking the popout button in Specviz displays "page not found by Solara server" at url `localhost:9000/solara//solara?kernelid=8ea216eb-64b9-4b96-b192-3a0cba895082&modelid=88022a5222164852860f8d175143c7ae&baseurl=/solara/&dark=true`.

Manually fixing the url to `localhost:9000/solara/?kernelid` correctly displays the page.  Changing the `baseurl` query parameter has no effect.  The `<body>` has tag `data-base-url="/solara/static/"`.  Changing this in the dev console does not change the popout url.  

How can I set `document.body` tags within the Solara app?    I need to set both the `data-base-url` and `data-voila-host` tags for ipypoput for different deploy environments.

**app.py**
```
from fastapi import FastAPI

import os
os.environ['SOLARA_ROOT_PATH'] = '/solara'
os.environ['SOLARA_APP'] = 'solara'
os.environ['SOLARA_THEME_VARIANT'] = 'dark'

import solara.server.fastapi

app = FastAPI()

@app.get("/")
def read_root():
    return {"Hello": "World"}

app.mount("/solara/", app=solara.server.fastapi.app)

```

**test.py**
```
import solara


@solara.component
def Page():
    with solara.Column():
        solara.Title("Spectral Display")

        from IPython.display import display
        import jdaviz
        from jdaviz import Specviz
        import ipysplitpanes
        import ipygoldenlayout
        import ipyvue
        import os
        from jdaviz.app import custom_components


        ipysplitpanes.SplitPanes()
        ipygoldenlayout.GoldenLayout()
        for name, path in custom_components.items():
            ipyvue.register_component_from_file(None, name, os.path.join(os.path.dirname(jdaviz.__file__), path))

        ipyvue.register_component_from_file('g-viewer-tab', "container.vue", jdaviz.__file__)

        solara.Info("Hello")
        with solara.Column():
            spec = Specviz()
            display(spec.app)
```

**Comment by iisakkirotko** [comment]
Hi! 

It looks to me like this is an issue with ipypopout. I took the liberty of opening an issue there (https://github.com/widgetti/ipypopout/issues/12), and will look to have this fixed. 

>How can I set document.body tags within the Solara app? I need to set both the data-base-url and data-voila-host tags for ipypoput for different deploy environments.

I don't think this is currently possible, since they are set by solara/voila themselves in templates, see 
https://github.com/widgetti/solara/blob/e16916ea3a120ec514e2170f8bed07e586b17191/solara/server/templates/solara.html.j2#L195

and 

https://github.com/voila-dashboards/voila/tree/9a7e6abc28bc1d397cc89825b4bb9ea15e21598a/share/jupyter/voila/templates

**Comment by havok2063** [comment]
Thanks for creating that issue.  I wasn't exactly sure which place to create it.   And just to clarify, the popout works correctly when running the Solara page by itself with "solara run test.py".  Likely as there's no additional base url.

**Comment by iisakkirotko** [comment]
@havok2063 Should be fixed in ipypopout 1.2.1, which was just released!

**Comment by havok2063** [comment]
@iisakkirotko I can confirm it works!  Thanks!
